dist/win-installer/mlpack-win-installer/Product.wxs

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <!-- DO NOT CHANGE the product GUID! It is forever -->
+	<Product Id="02A00C77-197D-4E91-B7D9-5836220E92E9"
+           UpgradeCode="6C2D7EC0-6F10-40CB-9703-1DC160A62662"
+           Name="MLPACK"
```

Comment:
Ah, just a minor issue, can we change `MLPACK` to `mlpack` to match the rest of the codebase? :)

Reply:
yep, done